wagn/lib/wagn/commands.rb

```diff
@@ -9,15 +9,15 @@ def load_rake_tasks
 
 RAILS_COMMANDS = %w( generate destroy plugin benchmarker profiler console server dbconsole application runner )
 ALIAS = {
-  "rs" => "rspec",
-  "cc" => "cucumber",
-  "jm" => "jasmine",
-  "g"  => "generate",
-  "d"  => "destroy",
-  "c"  => "console",
-  "s"  => "server",
-  "db" => "dbconsole",
-  "r"  => "runner"
+  'rs' => 'rspec',
+  'cc' => 'cucumber',
+  'jm' => 'jasmine',
+  'g'  => 'generate',
+  'd'  => 'destroy',
+  'c'  => 'console',
+  's'  => 'server',
+  'db' => 'dbconsole',
+  'r'  => 'runner'
```

Comment:
In case you're wondering.  The idea is 'strength reduction' for string constants.  If you don't have any double quote syntax, why incur any costs calling methods to parse.  Not that it is a good idea to be rigid about it or go through and change them all, but I think it's good as a standard style.